src/pint/models/astrometry.py

```diff
@@ -275,6 +286,10 @@ def get_psr_coords(self, epoch=None):
     def coords_as_ICRS(self, epoch=None):
         return self.get_psr_coords(epoch)
 
+    def coords_as_Eclip(self, epoch=None):
```

Comment:
For these names, I think I prefer either shorter e.g. `coords_as_ECL` or longer e.g. `coords_as_PulsarEcliptic` names.

Reply:
I will update with the next commit.